packages/zoe/src/zoeService/utils.d.ts

```diff
@@ -1,10 +1,5 @@
 import { Callable } from '@agoric/eventual-send';
-import type {
-  Instance,
-  IssuerKeywordRecord,
-  Payment,
-  StandardTerms,
```

Comment:
why didn't lint catch this?

Reply:
I looked into it and came up empty in the time I could allot. Since it's stripped out before runtime I think it's okay to let slide.